mtools/mlaunch/mlaunch.py

```diff
@@ -42,10 +42,10 @@
 
     from pymongo.errors import ConnectionFailure, AutoReconnect
     from pymongo.errors import OperationFailure, ConfigurationError
-except ImportError:
+except ImportError as e:
     raise ImportError("Can't import pymongo. See "
                       "http://api.mongodb.org/python/current/ for "
```

Comment:
Since the message is being updated, should also use the current url: `https://api.mongodb.com/python/current/`